File: 02_ProgrammingFundamentals/_04_Methods/_01_Lecture/_06_CalculateRectangleArea.java
package _02_ProgrammingFundamentals._04_Methods._01_Lecture;

import java.util.Scanner;

public class _06_CalculateRectangleArea {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        double width = Double.parseDouble(scan.nextLine());
        double height = Double.parseDouble(scan.nextLine());

        // Create method that calculates rectangle area
        rectangleArea(width, height);

        scan.close();
    }

    private static void rectangleArea(double width, double height) {
        System.out.printf("%.0f", width * height);
    }
}

File: 24_C#Fundamentals/_04_Methods/_01_Lecture/_06_CalculateRectangleArea/_06_CalculateRectangleArea.cs
using System;

public class _06_CalculateRectangleArea
{
    public static void Main(string[] args)
    {
        // Read input values
        double width = double.Parse(Console.ReadLine());
        double height = double.Parse(Console.ReadLine());

        // Create method that calculates rectangle area
        rectangleArea(width, height);
    }

    private static void rectangleArea(double width, double height)
    {
        Console.WriteLine($"{width * height:F0}");
    }
}
